config/v1/types_node.go

```diff
@@ -46,6 +46,17 @@ type NodeSpec struct {
 	// the status and corresponding reaction of the cluster
 	// +optional
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
+
+	// MinimumKubeletVersion is the lowest version of a kubelet that can meaningfully join the cluster.
```

Comment:
I"d suggest to avoid subjective language like meaningfully, be direct like our product docs.
```suggestion
	// minimumKubeletVersion is the lowest version of a kubelet that can join the cluster.
```